core/arch/arm/kernel/tee_ta_manager.c

```diff
@@ -1571,22 +1553,6 @@ void tee_ta_set_current_session(struct tee_ta_session *sess)
 	assert((sess == NULL) == !core_mmu_user_mapping_is_active());
 }
 
-TEE_Result tee_ta_get_client_id(TEE_Identity *id)
-{
```

Comment:
Please keep it if this is not an issue. This can be used to know whether or not  the client application is trusted or not.